components/table/InternalTable.tsx

```diff
@@ -117,6 +118,7 @@ export interface TableProps<RecordType = any>
   };
   sortDirections?: SortOrder[];
   showSorterTooltip?: boolean | TooltipProps;
+  sorterTooltipTarget?: SorterTooltipTarget;
```

Comment:
How about `showSorterTooltip={{ target: 'full-header' | 'sorterIcon' }}`

Reply:
Sounds good. I've asked @Ke1sy to make the suggested change. 

Nit: let's use the same naming convention (kebab case) for each option. So `showSorterTooltip={{ target: 'full-header' | 'sorter-icon' }}`.